### PR TITLE
New Json, New Dataset, Rename Triggers

### PR DESCRIPTION
**Pull request by DouglasBurns**
New Lumi Json - final of 2015 data.
New scaleup/down datsets
Rename Mu triggers to reflect no eta dependence
Update crab terminology inn makeconfig.py

### CODE REVIEW

File: python/submissionScripts/condor/makeConfig.py [hunk 1]
```diff
@@ -2,15 +2,16 @@
 import glob
 from mergeROOTFilesWithCompression import getGroupedFilesToUse
 
-pathOfCrabWorkdirs = '/storage/db0268/TopCrossSections/NTupleProduction/CMSSW_7_4_15/src/workdirCrab/v29/2015-10-28'
+pathOfCrabWorkdirs = '/storage/db0268/TopCrossSections/NTupleProduction/CMSSW_7_4_15/src/workdirCrab/v2/2015-11-10'
 
 version = pathOfCrabWorkdirs.split('/')[-2]
+print version
```
Comment (EmyrClement) [inline python/submissionScripts/condor/makeConfig.py]: Please remove print statement

File: python/submissionScripts/condor/makeConfig.py [hunk 2]
```diff
@@ -2,15 +2,16 @@
...
 
 ntupleBaseDir = '/hdfs/dpm/phy.bris.ac.uk/home/cms/store/user/doburns/'
 
 outputBaseDir = '/hdfs/TopQuarkGroup/run2/ntuples/25ns/'
 outputDirHdfs = outputBaseDir + version + '/'
-
+print outputDirHdfs
```
Comment (EmyrClement) [inline python/submissionScripts/condor/makeConfig.py]: Please remove print statement

**Comment by EmyrClement** [comment]
Not sure if suff in python/Doug belongs in the master branch...

**Comment by DouglasBurns** [comment]
Ah i tried to when combining some commits - didnt work - will remove in new
commit

*________________________*


*Douglas Burns*
*PhD Student*

School of Physics
University of Bristol
HH Wills Physics Lab
University of Bristol
Tyndall Avenue
Bristol
BS8 1TL

 Douglas.Burns@bristol.ac.uk <John.Newby@bristol.ac.uk>

**Pull request merged**